src/app/utils/boosts.js

```diff
@@ -223,10 +226,17 @@ export const boostCombinations = [
             'multiplier'
           ) || 1
 
-        const boostMultiplier =
-          prismMultiplier * boostBoost * (1 + (artifactBoostBoostBonus / 100))
+          var boostMultiplier;
+
+          if(artifactBoostBoostBonus)
+            boostMultiplier = prismMultiplier * boostBoost * (1 + (artifactBoostBoostBonus / 100));
```

Comment:
is the if-statement superfluous? artifactBoostBoostBonus is initialized to 0
`boostMultiplier = prismMultiplier * boostBoost * (1 + (artifactBoostBoostBonus / 100));`
should yield the correct value whether artifactBoostBoostBonus is zero or not.

Reply:
Heya! So I was testing an issue with a null value during testing - so, yes, this is 100% un-needed! Thanks :) Removed in latest commits, which'll be up soon!